src/styles/index.scss

```diff
@@ -49,6 +49,11 @@
 }
 
 .pgn__card-logo-cap {
```

Comment:
[unrelated nit] Do you mind adding a `TODO` comment above this Paragon class override, to suggest this override should likely be contributed upstream to Paragon (if it hasn't already)?